keyboards/angel64/alpha/readme.md

```diff
@@ -0,0 +1,15 @@
+# angel64
+
+![angel64](https://i.gyazo.com/6e2ea6c58d3253c496dc0518f2641ff9.jpg)
+
+Keyboard for tablets.
+
+Keyboard Maintainer: [kakunpc](https://github.com/kakunpc)  
```

Comment:
We recently changed the template to make this section a list, because it doesn't require any trailing spaces.

```suggestion
* Keyboard Maintainer: [kakunpc](https://github.com/kakunpc)
```